.gitignore

```diff
@@ -3,7 +3,7 @@ logs
 *.log
 npm-debug.log*
 
-lib
+#lib
```

Comment:
Hi Piotr! Thank you for your feedback!
It is because in my application here I am using "yarn add" directly from the github to another repository. But you are right, for the pull request I must uncomment it, I have just done so. :)